src/Juno/Consensus/Commit.hs

```diff
@@ -67,7 +67,7 @@ applyCommand :: Monad m => UTCTime -> Command -> Raft m (NodeID, CommandResponse
 applyCommand tEnd cmd@Command{..} = do
   apply <- view (rs.applyLogEntry)
   logApplyLatency cmd
-  result <- apply _cmdEntry
+  result <- apply (_cmdEntry, _cmdRequestId)
```

Comment:
hmmm... should we just be handing off the entire Command? the incremental add of "requestId" seems to suggest we might want other stuff in the future. What's more it took me a while to figure out what changed to add it to `applyFun`, `runCommand` etc, and is kind of "spec specific" now. Just dumping off the whole command seems more future-proof.